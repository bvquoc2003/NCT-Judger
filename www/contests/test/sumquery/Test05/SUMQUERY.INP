551
824
938
705
1003
930
281
269
873
987
785
350
1072
593
981
292
765
232
700
274
193
359
507
510
880
307
202
940
547
720
459
911
486
1013
892
845
423
688
1026
953
232
905
112
808
555
1003
890
721
912
939
359
188
803
539
869
990
112
745
825
526
428
878
229
300
469
883
132
212
652
578
787
616
941
444
858
611
1066
1092
891
831
557
1014
932
319
635
207
830
463
703
483
400
507
349
703
562
204
770
803
1010
443
305
711
672
259
338
625
758
807
611
266
129
494
154
257
107
631
887
282
806
918
417
181
198
106
188
1087
312
248
334
480
510
124
715
1064
841
948
621
232
804
1042
399
127
713
637
991
659
785
875
800
576
169
250
250
580
730
215
696
887
939
712
1053
941
748
643
336
537
677
329
487
271
460
149
154
642
415
770
307
359
444
1098
1077
820
796
813
365
947
1072
847
1062
1003
749
141
626
591
938
753
253
133
481
509
810
632
542
1074
491
162
559
208
890
738
696
537
369
1076
572
128
901
597
465
964
262
925
776
1062
836
957
159
347
808
711
576
1093
329
773
130
933
1018
206
551
1053
339
974
810
849
985
625
778
225
948
428
912
164
1093
991
600
231
396
922
709
290
321
792
481
778
755
1060
220
381
415
101
685
938
142
891
676
641
504
657
635
586
851
172
583
536
937
815
238
119
320
218
988
696
880
800
248
866
875
493
236
883
112
429
903
544
331
157
120
672
438
538
495
999
566
369
386
619
792
876
775
174
452
424
469
942
638
492
855
850
854
277
833
807
377
195
724
162
306
511
307
822
274
260
294
1061
1046
422
287
1066
264
957
1092
919
819
1027
963
579
1007
874
415
733
1095
974
859
1082
833
259
899
679
621
450
1062
996
656
415
307
334
561
465
139
559
977
525
293
920
967
835
885
838
560
383
632
965
517
795
525
176
520
957
146
340
597
503
316
787
432
844
886
899
700
889
556
127
399
191
210
715
502
1033
1026
782
110
1010
912
523
456
1080
915
966
811
751
480
728
1065
963
869
818
126
230
196
455
302
302
804
808
1057
253
250
874
902
143
683
1096
765
465
201
511
554
474
628
847
820
1051
255
216
482
456
966
358
815
103
674
233
926
496
246
380
956
842
233
761
439
640
614
838
884
845
337
263
258
568
960
752
514
726
984
271
294
783
860
523
784
420
1020
521
937
812
239
571
689
1080
511
376
102
980
866
231
590
235
779
343
618
657
937
112
125
993
922
547
220
1032
335
457
345
353
916
755
979
414
1068
1066
646
426
980
687
575
325
589
1064
715
616
756
445
104
281
442
406
461
5
142
443
539
379
394
398
523
231
479
89
496
69
375
217
267
244
366
492
505
474
498
78
519
197
445
107
312
21
224
32
426
48
391
9
128
426
431
529
544
131
288
143
373
29
461
427
467
202
289
344
525
63
444
518
538
321
469
182
403
12
300
111
384
387
499
305
356
409
486
384
486
358
465
135
495
15
55
34
409
309
395
254
331
370
485
139
329
21
292
170
516
487
525
486
487
14
339
162
550
256
380
3
487
336
536
191
479
492
550
501
535
363
531
372
529
299
397
467
507
6
274
486
516
55
350
361
468
100
172
532
548
131
152
275
497
526
546
327
412
38
514
451
459
475
541
42
385
491
521
412
536
292
495
316
440
398
532
372
508
390
392
180
185
19
379
170
230
391
475
34
378
370
542
271
463
334
456
442
520
338
482
112
326
202
421
407
437
326
346
389
493
70
203
269
510
18
167
547
547
374
522
371
441
