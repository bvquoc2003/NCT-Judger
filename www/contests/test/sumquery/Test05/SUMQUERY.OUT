100121
36169
81778
59070
11176
76295
153766
247272
187934
33588
75662
9449
15248
268906
155057
122489
124020
243356
210428
70805
4651
11034
98592
144976
267694
24981
54191
115398
234193
12907
94504
138832
175560
167250
68753
31610
47679
63684
68206
223922
25949
231310
56283
44749
71556
116974
165945
215452
23284
600
195375
241067
75903
297489
126094
178021
37271
20626
103513
95840
62349
24680
164216
18822
176458
68102
39244
11903
13622
136745
13637
55320
292916
5541
41534
210254
19068
76252
126172
80434
81271
83584
2114
4818
220009
38604
51195
211142
106638
119495
79020
47641
92228
128445
136622
20842
11524
63640
80344
149145
89383
1064
90818
43650
